.github/workflows/e2e-cleanup-weekly.yml

```diff
@@ -4,7 +4,6 @@ on:
   schedule:
     - cron: "0 0 * * 0" # At 00:00 every Sunday UTC
   workflow_dispatch:
-    
```

Comment:
I think the double newline is fine. Also omitting this doesn't add a commit to the file's history. So I think we should omit this for now.